docs/advanced-usage/user-quotas/index.md

```diff
@@ -0,0 +1,144 @@
+# User quotas
+
+Since
+[REANA 0.8.0](https://blog.reana.io/posts/2021/release-0.8.0/#cpu-and-disk-quota-accounting),
```

Comment:
Since
[REANA 0.8.0](https://blog.reana.io/posts/2021/release-0.8.0/#cpu-and-disk-quota-accounting), the cluster administrators have a possibility to set user quotas to limit the amount of disk space and CPU time available to execute workflows. This page explains how to inspect your limits and manage their use.

(taking user-oriented perspective, rather than admin-oriented perspective, in the narrative)